lib/Runtime/ByteCode/ByteCodeWriter.cpp

```diff
@@ -1681,6 +1681,8 @@ namespace Js
             case OpCode::StEnvSlotChkUndecl:
             case OpCode::StEnvObjSlot:
             case OpCode::StEnvObjSlotChkUndecl:
+            case OpCode::StModuleSlot:
+            case OpCode::LdModuleSlot:
```

Comment:
Just a future note. I expect we're going to want a Profiled variant of LdModuleSlot. Very easy change.